aces/Loris
# [Core] Add config option for projects to specify how they want dates to be displayed

## Brief summary of changes

Following today's meeting, this adds a config option to allow projects to specify how they want dates to be displayed in LORIS.

This PR only adds two quick examples of using this Config setting; it is by no means comprehensive.

Also, this will only take place in "PHP contexts" and not within JS date elements as these are OS- and locale-dependent.

## Testing

1. On aces/major go to the issue tracker module and look at the `Last Updated` column.
2. Checkout my branch and run the SQL patch.
3. Go to the configuration module and change the date format to something like `ymd`.
4. Reload the issue tracker module. The date format in the `Last Updated` column should match the format you supplied in step 3.
